docker-entrypoint.sh

```diff
@@ -329,12 +332,47 @@ liquibaseChangelockReset() {
   echo "End releasing liquibase changeloglock."
 }
 
+validatePostgresConnection() {
+  # https://stackoverflow.com/a/58784528/6921621
+    echo "Checking for postgres connectivity: "
+    if psql "${alertDatabaseConfig}" -c '\l';
+    then
+      echo "Alert postgres database connection valid."
+    else
+      echo "Alert postgres connection cannot be made."
+      sleep 10
+      exit 1
+    fi
+}
+
+createPostgresDatabase() {
+  # https://stackoverflow.com/a/58784528/6921621
```

Comment:
Same as above since it's the same link

Reply:
If we have an external database we will check to see if the alert database tables have been created in that database.  If they haven't then create the initial table schema into postgres.